### DISCUSSION TITLE
The Pro version is expired

### DISCUSSION THREAD

**Discussion by elena192020** [templaza/astroid-framework#888] [General]
Should I pay for the Pro version each year, or do you have a lifetime version?
What's happen when the Pro version is expired?
In this case could I use Pro add-ons?

**Comment by elena192020** [comment]
Do you have any videos how to use the Astroid template?

**Comment by Dnsftl** [comment]
Information about the price can be found here https://astroidframe.work/pricing

Documentation can be found here: https://docs.astroidframe.work/

Videos about Astroid can be found here: https://www.youtube.com/@astroidframework2932/videos

**Reply by elena192020** [reply]
I can't find a reply in above links about my question.
Should I pay for the Pro version each year, or do you have a lifetime version?
What's happen when the Pro version is expired?
In this case could I use Pro add-ons?

**Comment by sonvnn** [comment]
We do not offer lifetime subscription. With pro version you can keep and using it in the life-time. If your license is expired you can continue use your current version but you can not get update/maintain/support for it.

I hope it is clear enough for your question.

If you have any questions don't hesitate to post them here.

Best regards,
Sonny
